src/packagedcode/cargo.py

```diff
@@ -107,18 +108,90 @@ def build_package(package_data):
 
     name = package_data.get('package').get('name')
     version = package_data.get('package').get('version')
+    description = package_data.get('package').get('description')
+    if description:
+        description = description.strip()
 
     # TODO: Remove this ordered_dict_map once cargo.py is able to handle
     # the appropriate data (source_packages, dependencies, etc..)
     # At the moment, this is only useful for making tests pass
     ordered_dict_map = {}
```

Comment:
`ordered_dict_map` sounds a bit strange for a variable name: do not say what type it is, but name its content instead.
But also it might be much simpler to no create empty fields that are not needed and instead just add keyword arguments directly to your package constructor below:
```
authors = package_data.get('package', {}).get('authors', [])
# this assume a refactored party_mapper based on other comment 
parties = list(party_mapper(authors, party_type='author'))
package = RustCargoCrate(
    name=name,
    version=version,
    description=description,
    parties=parties,
)
```

Reply:
> `ordered_dict_map` sounds a bit strange for a variable name: do not say what type it is, but name its content instead.

Good variable naming conventions. I'll keep that it mind. :)

> But also it might be much simpler to no create empty fields that are not needed and instead just > add keyword arguments directly to your package constructor below:
> ...

I did try something like this in #1426 but for a reason I couldn't figure out - tests kept failing. Anyway, just tried to make it work this way again and seems like I had to also modify the respected fields in `Cargo.toml.expected` files from `{}` to `[]`.